opentelekomcloud/services/cce/addon-templates-v1.19.8.md

```diff
@@ -39,6 +39,8 @@ are no unneeded nodes.
 }
 ```
 
+See the [OTC Cloud Container Enging Addon Documentation](https://docs.otc.t-systems.com/en-us/usermanual2/cce/cce_01_0154.html) for more information.
```

Comment:
Cloud Container Engine

Reply:
Sure, I will fix it tomorrow with the other suggestion

Reply:
@DanielHabenicht will you continue here? :)

Reply:
thanks for the reminder.